### PR TITLE
UXD-1150 - Resolve some dependabot alerts

### PR DESCRIPTION
**Pull request by jamiek-acl**
### Purpose 🚀
Upgraded Lerna, Storybook, lint-staged, required Node version and some Babel plugins.

### Notes ✏️
1. No `changeset` necessary.
2. The Lerna upgrade is a major version bump, but [according to its author](https://github.com/lerna/lerna/releases/tag/v4.0.0): `Nothing was newly deprecated, nothing previously deprecated was removed. Migrating to v4 should be straightforward.` 
3. There is annoying console message that started showing up repeatedly when starting Storybook that I was unable to get rid of:

```
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
```

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -14,12 +14,9 @@ module.exports = function BabelConfigJS(api) {
         useESModules: false,
       },
     ],
-    [
-      "@babel/plugin-proposal-class-properties",
-      {
-        loose: true,
-      },
-    ],
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
```
Comment (nahumzs) [inline babel.config.js:17]: these were required after the upgrade I can imagine.

File: package.json [hunk 2]
```diff
@@ -82,8 +83,8 @@
     "jest-axe": "^4.1.0",
     "jest-cli": "^26.0.1",
     "js-yaml": "^3.13.1",
-    "lerna": "^3.13.0",
-    "lint-staged": "^8.2.1",
+    "lerna": "^4.0.0",
```
Comment (nahumzs) [inline package.json:86]: 👏

File: babel.config.js (hunk 1 above)
Comment (jamiek-acl) [reply babel.config.js:17]: Yes, Storybook needed them

**Review by nahumzs — APPROVED** [review]

**Comment by jamiek-acl** [comment]
This resolved 6 out of the 16 dependabot alerts.

**Pull request merged**